llpc/include/llpc.h

```diff
@@ -109,14 +109,26 @@ struct ShaderModuleBuildOut {
   ShaderModuleData *pModuleData; ///< Output shader module data (opaque)
 };
 
+enum StageCacheInfo : uint8_t {
+  Invalid = 0, ///< Stage cache miss/hit information is invalid.
```

Comment:
I don't know if I like "Invalid" for this name.  It gives the impression that something went wrong.  Something like "CacheUnchecked" might be better.

The comment should be clearer.  From this comment is it not clear why it should be invalid.  You could something like "The cache was not checked."

Reply:
done.